src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -125,4 +132,28 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes(RouteCollection $routes, string $propertyName): RouteCollection
+    {
+        $validOptions = ['', 'priority', 'name', 'path'];
+        $sortedRoutes = $routes->all();
+        if ('name' === $propertyName) {
+            ksort($sortedRoutes);
+        } elseif ('path' === $propertyName) {
+            uasort(
```

Comment:
this would be as effective:
`uasort($sortedRoutes, function ($a, $b) { return $a->getPath() <=> $b->getPath(); })`

Reply:
PHPCS is not happy about this, but if fabbot doesn't raise a notice I'll keep it as you suggested, thanks!